Makefile.am

```diff
@@ -353,10 +353,9 @@ dist-hook::
 	( cd $(srcdir); git ls-tree HEAD --name-only -r $(COMMITTED_DIST) || (echo $(COMMITTED_DIST) | tr ' ' '\n' ) ) | \
 		tar -C $(srcdir) -cf - -T - | tar -C $(distdir) -xf -
 	tar -C $(srcdir) -cf - --exclude='phantomjs*' node_modules/ | tar -C $(distdir) -xf -
-	tar -C $(srcdir) -cf - bower_components/ | tar -C $(distdir) -xf -
 	echo $(VERSION) > $(distdir)/.tarball
-	$(srcdir)/tools/build-copying $(BOWER) > $(distdir)/COPYING.bower
-	[ ! -d $(distdir)/tools/debian ] || $(srcdir)/tools/build-debian-copyright $(BOWER) > $(distdir)/tools/debian/copyright
+	$(srcdir)/tools/build-copying $(distdir)/bower_components > $(distdir)/COPYING.bower
+	[ ! -d $(distdir)/tools/debian ] || $(srcdir)/tools/build-debian-copyright $(distdir)/bower_components > $(distdir)/tools/debian/copyright
```

Comment:
Oh and yes, it's intentional that this is now using ```$(distdir)``` since all licenses that are to be included in the distribution are copied there. Not all licenses in ```$(srcdir)``` should be considered. Again see #6814 for reasoning here ... since we start looking at ```node_modules``` instead of ```bower_components``` and we certainly don't dist or include licenses for all node modules.